packages/create-gasket-app/lib/utils/warn-if-outdated.js

```diff
@@ -0,0 +1,76 @@
+const { runShellCommand } = require('@gasket/utils');
+const semver = require('semver');
+const chalk = require('chalk');
+const { readFile, writeFile } = require('fs/promises');
+const path = require('path');
+
+const cachePath = path.join(__dirname, '..', '..', '.cache'); // Place at root of gasket-cli
```

Comment:
I think the `.cahe` file is getting added to the root of the `create-gasket-app` directory with this change and not to the `gasket-cli` directory. Not sure if we still want the `.cache` file to be saved to the `gasket-cli` directory or we are moving it to the `create-gasket-app` directory. 

If we keep `.cache` here we need to update the `.gitignore` to ignore the new cache path.

Reply:
Just realized that `warn-if-outdated` is duplicated in `create-gasket-app` and `gasket-cli`. 

Maybe it's worth a ticket to consolidate this file in one place and share that between the two packages? @agerard-godaddy @mmason2-godaddy

Reply:
You shouldn't need the warning if outdated at all right? Although the naming of the CLI in the index is `gasket` this is essentially a completely separate CLI.

Reply:
@jpina1-godaddy Let's create a ticket to clean up the duplicate `/utils` folder. Not sure where is should go exactly.... maybe have `create-gasket-app` depend on `@gasket/cli` and update the exports of the CLI package.

Reply:
Sounds good. I created a ticket in the backlog to give this issue a little more thought and to potentially remove this from `create-gasket-app` if it's not needed.

Reply:
Awesome thank you. It may be needed but it's a weird edge case. If you every `npm i -g create-gasket-app` then you'd want it so we probably should have it given that's possible scenario. The ticket to create exported utils from the CLI is still relevant.